types/validator/validator-tests.ts

```diff
@@ -924,6 +922,11 @@ const any: any = null;
     result = validator.toInt(any);
     result = validator.toInt(any, 10);
 }
+{
+    let result: boolean|number;
+    result = validator.isStrongPassword("sample");
+    result = validator.isStrongPassword("sample23#@test");
```

Comment:
this is not correct IMO, see my comment about return type. This require scoring function switch IMO.